Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Resources/settings.json

```diff
@@ -93,7 +93,8 @@
     "timeoutScript": 10,
```

Comment:
discussed, file to be removed/merged into the Aquality.Selenium.Template/Resources/settings.json and settings.azure.json

Reply:
This file was removed.